pkg/pb/synthetic_monitoring/checks.proto

```diff
@@ -575,3 +578,18 @@ message GrpcSettings {
   bool                      tls             = 3 [(gogoproto.jsontag) = "tls,omitempty"];
   TLSConfig       tlsConfig           = 4 [(gogoproto.jsontag) = "tlsConfig,omitempty"];
 }
+
+// CheckStats provides information about the executed checks.
+message CheckStats {
+  string                      instance           = 1 [(gogoproto.jsontag) = "instance"];
```

Comment:
This is necessary to avoid mixing up stats from different agent instances. If these were regular Prometheus metrics, this would be added by the scraper.